app/controllers/provider_interface/interviews_controller.rb

```diff
@@ -70,18 +54,18 @@ def commit
         redirect_to provider_interface_application_choice_interviews_path(@application_choice)
       else
         track_validation_error(@wizard)
-        render :check
+        render 'new'
```

Comment:
Is there a reason this is a string now? Will this render the check page, or jump back to the form view?

Reply:
No reason it's a string. I was trying to redirect and that wasn't working as expected so I switched back to rendering partials, which is why I didn't reuse the existing format. Happy to change.

It will switch back to the form view as that's the one place where any errors can be highlighted to the user.